cmd/format.go

```diff
@@ -396,6 +396,9 @@ func format(c *cli.Context) error {
 			} else {
 				logger.Warnf("List storage %s failed: %s", blob, err)
 			}
+			if err = blob.Put("juicefs_uuid", strings.NewReader(format.UUID)); err != nil {
```

Comment:
Make sure it's deleted with destroy command.

Reply:
it will, as destroy cleans up the whole blob, including testing, chunk, meta and this new object.